src/Service/ExcelExport.php

```diff
@@ -167,15 +164,13 @@ public function generateExcelFile(LsDoc $cfDoc, array $items, array $association
         $activeSheet = $phpExcelObject->setActiveSheetIndex(2);
         $activeSheet
             ->setCellValue('A1', 'identifier')
-            ->setCellValue('B1', 'uri')
-            ->setCellValue('C1', 'originNodeIdentifier')
-            ->setCellValue('D1', 'originNodeUri')
-            ->setCellValue('E1', 'destinationNodeIdentifier')
-            ->setCellValue('F1', 'destinationNodeUri')
-            ->setCellValue('G1', 'associationType')
-            ->setCellValue('H1', 'associationGroupIdentifier')
-            ->setCellValue('I1', 'associationGroupName')
-            ->setCellValue('J1', 'lastChangeDateTime')
+            ->setCellValue('B1', 'originNodeIdentifier')
+            ->setCellValue('C1', 'originNodeHumanCodingScheme')
+            ->setCellValue('D1', 'associationType')
+            ->setCellValue('E1', 'destinationNodeHumanCodingScheme')
+            ->setCellValue('F1', 'destinationNodeIdentifier')
```

Comment:
Note that without allowing the destination URI an exemplar cannot really be seen nor edited (since the URI is what holds what the exemplar is pointing to)